### PR TITLE
Review and Edit on ANN.md by salmaan1234

### PR DESCRIPTION
**Pull request by Nugetzrul3**
-Few grammar changes
-Change solo mining directions. Please check and confirm :)

salmaan1234

### CODE REVIEW

File: doc/ANN.md [hunk 1]
```diff
@@ -171,4 +173,4 @@ information or see https://opensource.org/licenses/MIT.
 Community
 ---------
 - Discord: https://discord.gg/rm7HHv6
-- Twitter: https://twitter.com/sugarchain_dev
\ No newline at end of file
+- Twitter: https://twitter.com/sugarchain_dev
```
Comment (cryptozeny) [inline doc/ANN.md]: not changed but why its changing?

File: doc/ANN.md [hunk 2]
```diff
@@ -118,11 +118,13 @@ Wallet
...
 --------
 Bech32 address is by default and strongly recommended. `-t1` uses 1 thread. If you want more hash, increase this number.
-- Solo mining: Make a file `sugarchain.conf`, restart your wallet and run cpuminer-opt-sugarchain (RPC=`34229`, testnet5 RPC=`44229`, regtest RPC=`45339`)
+- Solo mining: Open wallet and go to Settings>Options>Wallet Configuration and enter the code below , restart your wallet and run cpuminer-opt-sugarchain (RPC=`34229`, testnet5 RPC=`44229`, regtest RPC=`45339`)
 ```
 server=1
 rpcuser=username
 rpcpassword=password
+rpcallowip=127.0.0.1
+rpcport=34229
```
Comment (cryptozeny) [inline doc/ANN.md]: rpcport is already have. not needed in general.

Comment (Nugetzrul3) [reply doc/ANN.md]: Ye ik but its for like copy and paste the whole thing

File: doc/ANN.md [hunk 3]
```diff
@@ -118,11 +118,13 @@ Wallet
 Cpuminer
 --------
 Bech32 address is by default and strongly recommended. `-t1` uses 1 thread. If you want more hash, increase this number.
-- Solo mining: Make a file `sugarchain.conf`, restart your wallet and run cpuminer-opt-sugarchain (RPC=`34229`, testnet5 RPC=`44229`, regtest RPC=`45339`)
+- Solo mining: Open wallet and go to Settings>Options>Wallet Configuration and enter the code below , restart your wallet and run cpuminer-opt-sugarchain (RPC=`34229`, testnet5 RPC=`44229`, regtest RPC=`45339`)
 ```
 server=1
 rpcuser=username
 rpcpassword=password
+rpcallowip=127.0.0.1
```
Comment (cryptozeny) [inline doc/ANN.md]: i think rpcallowip is not needed? could you check it again?

File: doc/ANN.md (hunk 1 above)
Comment (Nugetzrul3) [reply doc/ANN.md]: Idk lol but its the same thing

File: doc/ANN.md (hunk 3 above)
Comment (Nugetzrul3) [reply doc/ANN.md]: I will check when i get home and test whether miner works without allowip

**Pull request merged**